maestrowf/interfaces/script/slurmscriptadapter.py

```diff
@@ -344,21 +353,24 @@ def _state(self, slurm_state):
         :returns: A Study.State enum corresponding to parameter job_state.
         """
         LOGGER.debug("Received SLURM State -- %s", slurm_state)
-        if slurm_state == "R":
+        if slurm_state == "R" or slurm_state == "RUNNING":
```

Comment:
Can these actually be abbreviated in `sacct`?

Reply:
I'll play the format options and see, but it's not clear that restricting state to 2 letters will actually do the same thing since running is only a 1 letter abbreviation.

Reply:
Sorry -- I was unclear. My question was whether or not we need the `or` clause in there.

Reply:
Though -- if we fall back to `squeue` if `sacct` doesn't work, then we'd need those there.